composer.json

```
@@ -26,24 +26,26 @@
     "psr/clock": "^1.0",
     "psr/container": "^2.0",
     "psr/event-dispatcher": "^1.0",
+    "psr/http-client": "^1.0",
     "psr/http-factory": "^1.1",
     "psr/http-message": "^2.0",
     "psr/log": "^1.0 || ^2.0 || ^3.0",
     "symfony/finder": "^6.4 || ^7.3 || ^8.0",
     "symfony/uid": "^6.4 || ^7.3 || ^8.0"
   },
   "require-dev": {
+    "laminas/laminas-httphandlerrunner": "^2.12",
+    "nyholm/psr7": "^1.8",
+    "nyholm/psr7-server": "^1.1",
     "php-cs-fixer/shim": "^3.91",
     "phpstan/phpstan": "^2.1",
     "phpunit/phpunit": "^10.5",
     "psr/cache": "^3.0",
     "psr/simple-cache": "^3.0",
     "symfony/cache": "^6.4 || ^7.3 || ^8.0",
     "symfony/console": "^6.4 || ^7.3 || ^8.0",
-    "symfony/process": "^6.4 || ^7.3 || ^8.0",
-    "nyholm/psr7": "^1.8",
-    "nyholm/psr7-server": "^1.1",
-    "laminas/laminas-httphandlerrunner": "^2.12"
+    "symfony/http-client": "^7.4",
```

would need a version range as well I'd say - at least to be consistent
```suggestion
    "symfony/http-client": "^6.4 || ^7.3 || ^8.0",
```

---

Oh true true!